tools/ci_build/github/linux/docker/Dockerfile.ubuntu_cuda11_4_tensorrt7_2

```diff
@@ -0,0 +1,37 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with TensorRT integration
+
+ARG OS_VERSION=18.04
+FROM nvidia/cuda:11.4.0-cudnn8-devel-ubuntu${OS_VERSION}
+
+ENV LD_LIBRARY_PATH /usr/local/cuda-11.1/targets/x86_64-linux/lib/:$LD_LIBRARY_PATH
```

Comment:
cuda 11.3 works fine with cuda 11.4 but cuda 11.1 causes: 

Failed to load library libonnxruntime_providers_tensorrt.so with error: libnvrtc.so.11.1: cannot open shared object file: No such file or directory because  libnvrtc.so.11.1 only exists in cuda-11.1 (libonnxruntime_providers_tensorrt.so doesn't require this in TRT 8 + CUDA 11.3). 

Can find better solution next PR.